modules/cnv-vm-wizard-fields-web.adoc

```diff
@@ -42,37 +34,43 @@ endif::[]
 |Container
 |Provision virtual machine from a bootable operating system container located in a registry accessible from the cluster. Example: `_kubevirt/cirros-registry-disk-demo_`.
 
-|Cloned Disk
-|Provision source is a cloned disk.
+|Disk
+|Provision virtual machine from a disk.
 
-|Import
-|Import virtual machine from a supported provider.
+|Attach disk
+|
+|Provision an existing disk that was previously cloned or created and made available in the Persistent Volume Claims. When this option is selected, the operating system, flavor, and workload profile fields must be filled manually.
 
 |Operating System
 |
-|A list of operating systems available in the cluster. This is the primary operating system for the virtual machine. If you select `Import` as the *Provider Source*, the operating system is filled in automatically, based on the operating system of the VMware virtual machine being imported.
+|The primary operating system that is selected for the virtual machine.
 
 |Flavor
 |small, medium, large, tiny, Custom
-|Presets that determine the amount of CPU and memory allocated to the virtual machine.
+|Presets that determine the amount of CPU and memory allocated to the virtual machine. The presets displayed for Flavor are determined by the operating system.
 
 .3+|Workload Profile
-|desktop
+|High Performance
+|A virtual machine configuration that is optimized for high-performance loads.
```

Comment:
s/loads/workloads

Reply:
Modified "loads" to "workloads"
A virtual machine configuration that is optimized for high-performance workloads.